LISEZMOI.md

```diff
@@ -1,17 +1,33 @@
+
+## Structure du dépôt central
+
+Chaque type d’exécution d’OpenQuake a son propre dossier : dommages classiques (« cDamage »), danger classique (« cHazard »), risque basé sur les événements (« ebRisk »), et dommages stochastiques (« eDamage »). Veuillez noter que le CanadaSRM1 *ne comprend pas* les évaluations classiques des dangers ou des dommages. Les dossiers existants « cDamage » et « cHazard » ne font pas partie du lancement officiel, mais constituent plutôt des travaux en cours (bêta). Dans chacun des dossiers de type d’exécution se trouvent des scripts utilisés pour exécuter l’évaluation, ainsi que des dossiers d’entrée et de sortie. 
+
+Les scripts utilisés pour initialiser le modèle sont stockés dans « model-scripts », et les autres scripts se trouvent dans « scripts ». Certains documents pertinents se trouvent dans « documentation », et « docs », « data », et « admin » sont liés à l’administration de ce dépôt.
+
+## Utilisation technique
+
+Il est possible de télécharger des fichiers individuels à partir du Web, en cliquant sur « Données brutes » ou « Télécharger » dans le coin supérieur droit de la page d’aperçu d’un fichier. Si vous souhaitez cloner ce dépôt, veuillez vous assurer que [le logiciel Git LFS](https://git-lfs.github.com/) fonctionne sur votre appareil avant de procéder au clonage. Les évaluations des aléas sismiques présentées ici sont réalisées à l’aide de [l’OpenQuake Engine](https://www.globalquakemodel.org/openquake) qui peut être cloné [ici](https://github.com/gem/oq-engine). Tout fichier d’entrée stocké hors de ce dépôt sera rendu public dès que la documentation de référence sera disponible, y compris le modèle de source de danger, la réponse du site, l’exposition, la vulnérabilité et les fonctions de fragilité. 
 
 ## Documentation
-* Thumbnail with accompanying short description and links to the GSC OpenFile report to be published through OpenDRR GitHub. Consider including links to the OpenFile report that jump to relevant sections
-* Thumbnails with accompanying short description and links to technical documentation of the OpenQuake-engine
 
-## Entrées du modèle
-* Thumbnail with accompanying short description and links to relevant oq-model-input files used to assess probabilistic seismic risk (exposure & site models)
-* Curently using git-LFS to track model outputs in .csv, .gdp and gpkg formats
+Un rapport de dossier public de la Commission géologique du Canada contient des renseignements de base sur la façon dont ces modèles sont produits et fournit quelques exemples de sorties. Ce document se trouve auprès de [GEOSCAN](https://geoscan.nrcan.gc.ca/). Les articles de journaux à venir contiendront des renseignements supplémentaires et pourront être recherchés dans le [profil Google Scholar]] de T.E. Hobbs (https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao). 
```

Comment:
The following segment:

```markdown
... dans le [profil Google Scholar]] de T.E. Hobbs (https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao).
```

is misformatted and would be rendered as:

> ... dans le [profil Google Scholar]] de T.E. Hobbs (https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao).

I recommend changing the above to:

```markdown
... dans [le profil Google Scholar de T.E. Hobbs](https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao).
```

so it would be rendered properly like so:

> ... dans [le profil Google Scholar de T.E. Hobbs](https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao).

or, to be consistent with the current English version:

```markdown
... dans [le profil Google Scholar](https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao) de T.E. Hobbs.
```

> ... dans [le profil Google Scholar](https://scholar.google.ca/citations?user=0hqT-owAAAAJ&hl=en&oi=ao) de T.E. Hobbs.